sites-enabled/03-all-guide.org.conf

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------
+#  redirect guide to .org/, .org/forum
+# ------------------------------------
+
+# map for guide to other
+# subdomains
+map $request_uri $guide_to_other_uri {
+  include /etc/nginx/maps/guide-to-other.map;
+}
+
+# map for guide URLs
+map $request_uri $guide_uri {
+  include /etc/nginx/maps/guide.map;
+  default $request_uri;
+}
+
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name guide.freecodecamp.org;
+
+  # SSL
+  include snippets/common/ssl-freecodecamp-org.conf;
+
+  if ($guide_to_other_uri) {
+    return 302 $guide_to_other_uri;
+  }
+
+  return 302 https://www.freecodecamp.org$guide_uri;
+}
+
+# HTTP redirect
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name guide.freecodecamp.org;
+
+  if ($guide_to_other_uri) {
+    return 302 $guide_to_other_uri;
+  }
```

Comment:
Can you confirm this does not need the full URL?

Reply:
Yes, this works in my testing.

I left the `server_name guide.freecodecamp.org guide.freecodecamp.dev;` line up on the dev server, so if you go to https://guide.freecodecamp.dev/miscellaneous/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/ it should redirect to https://contribute.freecodecamp.org/.